src/platform/site-wide/onclicks.js

```diff
@@ -0,0 +1,15 @@
+/**
+ * Attaches onclick events to accordions links for GA tag tracking
+ */
+export function addClickEvent() {
+  const links = Array.from(
+    document.querySelectorAll('div.service-accordion-output a'),
+  );
+  for (const link of links)
+    link.setAttribute(
+      'onclick',
+      "recordEvent({ event: 'nav-accordion-embedded-link-click' })",
+    );
+}
+
+document.addEventListener('DOMContentLoaded', addClickEvent);
```

Comment:
Much better 😄 My only suggestion it to rename the file something a little more descriptive, maybe something like `wysiwyg-analytics-setup` or something along those lines? Or something to clear up that we're doing it this way because the content we want to add analytics to doesn't live in vets-website

Reply:
Makes sense to me, @ncksllvn - updated with new name.